src/webui/www/private/preferences_content.html

```diff
@@ -817,6 +817,263 @@
     </fieldset>
 </div>
 
+<div id="AdvancedTab" class="PrefTab invisible">
+    <fieldset class="settings">
+        <legend>QBT_TR(qBittorrent Section)QBT_TR[CONTEXT=OptionsDialog]&nbsp;(<a href="https://github.com/qbittorrent/qBittorrent/wiki/Explanation-of-Options-in-qBittorrent#Advanced" target="_blank">QBT_TR(Open documentation)QBT_TR[CONTEXT=HttpServer]</a>)</legend>
+        <table>
+            <tr>
+                <td>
+                    <label for="network_interface">QBT_TR(Network Interface (requires restart):)QBT_TR[CONTEXT=OptionsDialog]</label>
+                </td>
+                <td>
+                    <select id="network_interface" style="width: 15em;">
```

Comment:
You should be using camelcase for id.
IMO the only allowable place to use underscores is the settings/options key (when communicating with the server).

Reply:
Sorry but all of the ids in this file use underscore, none of them use camelcase. I just want to keep the consistency.

Reply:
>Sorry but all of the ids in this file use underscore, none of them use camelcase. 

Seems not true: 
https://github.com/qbittorrent/qBittorrent/blob/60974247d859eb67463cd563c2bb9a7777389504/src/webui/www/private/preferences_content.html#L737

IMO the only exception to keep using underscore naming in current commit is you'll add another commit that change all ids to camelcase... Everyone has to follow the coding gudieline.